bootstrap/kubeadm/config/manager/manager_image_patch.yaml

```diff
@@ -7,5 +7,5 @@ spec:
   template:
     spec:
       containers:
-        - image: gcr.io/k8s-staging-cluster-api/kubeadm-bootstrap-controller:master
+        - image: gcr.io/k8s-staging-cluster-api/kubeadm-bootstrap-controller-amd64:dev
```

Comment:
Was this change meant to be included?

Reply:
no, that was a mistake, thanks for pointing out